documentation/modules/ref-python-example.adoc

```diff
@@ -37,3 +37,28 @@ class HelloWorld(MessagingHandler):
 Container(HelloWorld("amqps://_messaging-route-hostname_:443", "myqueue")).run()
 ----
 
+== Known issue with creating a subscriber on a hierarchical topic
+
+A known issue exists where creating a subscriber on a hierarchical topic in {ProductName} causes the broker to instead create it as a competing consumer (handling the address like a queue rather than a topic).
+
+The workaround for this issue involves setting the capability `"topic"` in the source.
+
+.Procedure
+
+. In the `simple_recv.py` file, modify the `from proton.reactor import Container` to add the `ReceiverOption`:
```

Comment:
Shouldn't it be a bullet if it's only 1 item?

Reply:
Done